### PR TITLE
Add EVM version of Cronos

### PR DESCRIPTION
**Pull request by jamesbayly**

### CODE REVIEW

File: Cronos/cronos-evm-starter-via-eth/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# SubQuery - Starter Package
```
Comment (deverka) [inline Cronos/cronos-evm-starter-via-eth/README.md:1]: @jamesbayly this is the old version of readme

File: Cronos/cronos-evm-starter-via-eth/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+type Approval @entity {
+  id: ID! # Transaction hash
+  value: BigInt!
+
+  owner: String!
+
+  spender: String!
+
+  contractAddress: String!
```
Comment (deverka) [inline Cronos/cronos-evm-starter-via-eth/schema.graphql:24]: I would remove empty lines.

File: Cronos/cronos-evm-starter-via-eth/project.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      handlers:
+        - handler: handleTransaction
+          kind: ethereum/TransactionHandler
+          filter:
+            ## The function can either be the function fragment or signature
+            ## https://docs.ethers.io/v5/api/utils/abi/fragments/#FunctionFragment
+            # function: '0x095ea7b3'
+            # function: '0x7ff36ab500000000000000000000000000000000000000000000000000000000'
+            function: approve(address spender, uint256 rawAmount)
+        - handler: handleLog
```
Comment (deverka) [inline Cronos/cronos-evm-starter-via-eth/project.yaml:44]: Why only LogHandler? Is this simpler than using also ethereum/TransactionHandler?

File: Cronos/cronos-evm-starter-via-eth/project.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+repository: "https://github.com/subquery/cosmos-subql-starter/tree/main/Cronos/cronos-evm-starter-via-eth"
+schema:
+  file: ./schema.graphql
+network:
+  chainId: "25"
+  # This endpoint must be a public non-pruned archive node
+  # Public nodes may be rate limited, which can affect indexing speed
+  # When developing your project we suggest getting a private API key
+  # Take a look here for some Cronos endpoints https://moonflow.solutions/rpc-servers
+  endpoint: "https://evm.cronos.org/"
```
Comment (stwiname) [inline Cronos/cronos-evm-starter-via-eth/project.yaml:22]: We should be able to use the same dictionary from the cosmos sdk

Comment (stwiname) [reply Cronos/cronos-evm-starter-via-eth/project.yaml:22]: Ignore that. The dictionary is compatible, but the chainIds wont match. This is something we can improve in the sdk

Comment (deverka) [reply Cronos/cronos-evm-starter-via-eth/project.yaml:22]: Ok!

**Pull request merged**